contracts/monitoring/src/msg.rs

```diff
@@ -0,0 +1,16 @@
+use connection_router_api::ChainName;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use multisig::worker_set::WorkerSet;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub enum ExecuteMsg {}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(Vec<(ChainName, WorkerSet)>)]
+    GetChainActiveWorkerSet { chains: Vec<ChainName> },
```

Comment:
I think we should change worker to verifiers here, since we are now calling them verifiers. We should do this across the codebase, but definitely should not add new code that still uses workers.
```suggestion
    GetChainActiveVerifiers { chains: Vec<ChainName> },
```